doc/release-notes/10093-signedUrl_improvements.md

```diff
@@ -0,0 +1,5 @@
+A new version of the standard Dataverse Previewers from https://github/com/gdcc/dataverse-previewers is available. The new version supports the use of signedUrls rather than API keys when previewing restricted files (including files in draft dataset versions). Upgrading is highly recommended.
```

Comment:
I'm a little confused by this. When Dataverse 6.1 is released, will Dataverse Previewers 1.4 be released?

Reply:
That is my intent. Feel free to hedge.